package/kernel/linux/modules/netdevices.mk

```diff
@@ -1348,3 +1348,18 @@ define KernelPackage/sfc-falcon/description
 endef
 
 $(eval $(call KernelPackage,sfc-falcon))
+
+define KernelPackage/aqc
+  SUBMENU:=$(NETWORK_DEVICES_MENU)
+  TITLE:=Aquantia AQtion (AQC107) 10Gbps Ethernet NIC
+  DEPENDS:=@PCI_SUPPORT +kmod-ptp +kmod-hwmon-core
+  KCONFIG:= \
```

Comment:
You have to specify the Kconfig symbol and no `\` if there is not additional Kconfig symbol in the next line.

Reply:
@hauke thanks - can you please recheck and resolve if correct?